internal/pubsub/mempubsub/mem_test.go

```diff
@@ -75,3 +76,18 @@ func TestReceive(t *testing.T) {
 		t.Fatalf("got %d, want %d", got, want)
 	}
 }
+
```

Comment:
Fine for now, but we should stop adding mempubsub-specific tests and add them to `drivertest.go` instead. Someone implementing a new driver should ~not have to write tests for it.